src/content/contributing/release-process/_index.en.md

```diff
@@ -4,154 +4,225 @@ date: 2020-03-18T16:03:26+01:00
 weight: 10
 ---
 
-This section explains the necessary steps to make a submariner release.
-It is assumed that you are familiar with the submariner project and the various repositories.
+This section describes how to create a Submariner product release. It is assumed that you are familiar with the various
+Submariner projects and their repositories and are familiar with Git and GitHub.
 
+## Project dependencies
 
-## Step 1: Create a Submariner Release
+The Submariner projects have a dependency hierarchy with respect to their Go libraries and Docker images.
+Therefore the releases for each individual project must be created in a specific order.
 
+The Go dependency hierarchy flow is as follows:
 
-Assuming that you have an existing submariner git directory, the following steps create a release named "Globalnet Overlapping IP support RC0" with version v0.2.0-rc0 based on the master branch.
+`shipyard` <- `admiral` <- `[submariner, lighthouse]` <- `submariner-operator`
 
-```bash
-cd submariner
-git stash
-git remote add upstream ssh://git@github.com/submariner-io/submariner
-git fetch -a -v -t upstream
-git checkout remotes/upstream/master -B master
-git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
-git push upstream v0.2.0-rc0
-```
+Note that the `submariner` and `lighthouse` projects are siblings and thus do not depend on one another. Also the
+`submariner-operator` components expect that `lighthouse` and `submariner` are aligned on the same exact version.
 
-A tagged release should appear [here](https://github.com/submariner-io/submariner/tags).
+The Docker image dependency hierarchy flow is as follows:
 
-> https://github.com/submariner-io/submariner/tags
+`subctl binary` <- `shipyard dapper base image` <- `[admiral, submariner, lighthouse, submariner-operator]`
 
-A build for v0.2.0-rc0 should start and appear under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/submariner/branches).
+The dapper base image that is provided by `shipyard` for building and E2E tests in all of the other projects pulls in the
```

Comment:
I think "Dapper" should be capitalized in most cases here, where it's not part of some `code literal`.